CHANGELOG_UNRELEASED.md

```diff
@@ -1,3 +0,0 @@
-# Changelog (unreleased)
-
-/!\ This file should no longer be used, c.f., `doc/changelog/README.md` /!\
```

Comment:
@hoheinzollern @CohenCyril was this warning removed on purpose? maybe it's still a bit early? (c.f. https://github.com/math-comp/math-comp/pull/1296 for instance)

Reply:
Yes, that was removed on purpose, perhaps @CohenCyril we should discuss whether the removal is actually beneficial.

Reply:
> Yes, that was removed on purpose, perhaps @CohenCyril we should discuss whether the removal is actually beneficial.

I don't thing there is a discussion too have: it was too soon.